src/server/qgsserver.cpp

```diff
@@ -397,7 +397,7 @@ void QgsServer::handleRequest( QgsServerRequest &request, QgsServerResponse &res
         // Project is mandatory for OWS at this point
         if ( ! project )
         {
-          throw QgsServerException( QStringLiteral( "Project file error" ) );
+          throw QgsServerException( QStringLiteral( "Project file error. For OWS services: please provide a SERVICE and a MAP parameter pointing to a valid QGIS project file" ) );
```

Comment:
Maybe also specify that the project can be set from the command line with `-p <path to project>`

Reply:
Forget my comment. I thought we were still in the development server code part.